dali/sasha/saarch.cpp

```diff
@@ -1433,7 +1434,7 @@ class CSashaArchiverServer: public ISashaServer, public Thread
 
     void runArchiver(CBranchArchiver &archiver,Owned<IRemoteConnection> &conn)
     {
-        if (!stopped&&archiver.ready()) {
+        if (!stopped) {//if (!stopped&&archiver.ready()) {
```

Comment:
restore or remove the commented out code.

Reply:
whoops, I'll restore